tools/tertiary-analysis/scanpy/scanpy-neighbours.xml

```diff
@@ -38,13 +38,13 @@ PYTHONIOENCODING=utf-8 scanpy-neighbours.py
       <param name="default" type="boolean" checked="true" label="Use programme defaults"/>
       <when value="true"/>
       <when value="false">
-        <param name="n_neighbour" argument="--n-neighbors" type="integer" value="15" label="Maximum number of neighbours used"/>
+        <param name="n_neighbours" argument="--n-neighbors" type="integer" value="15" label="Maximum number of neighbours used"/>
```

Comment:
Slight inconsistency between wrappers and scanpy-scripts, but reasonable to leave it like it is.